module/VuFind/src/VuFind/Service/Factory.php

```diff
@@ -109,9 +109,11 @@ public static function getCart(ServiceManager $sm)
             ? (bool)$config->Site->showBookBag : false;
         $size = isset($config->Site->bookBagMaxSize)
             ? $config->Site->bookBagMaxSize : 100;
+        $activeInSearch = isset($config->Site->bookbagTogglesInSearch)
+            ? $config->Site->bookbagTogglesInSearch : true;
```

Comment:
Related to config.ini comment above, I see that you are defaulting this to true. I don't necessarily object to that, but I wanted to make sure it was intentional -- and if so, perhaps to discuss this one more time on a dev call before merging.